frontend/public/actions/ui.ts

```diff
@@ -124,7 +125,7 @@ export const getNamespacedResources = () => {
 };
 
 export const getActiveNamespace = (): string => store.getState().UI.get('activeNamespace');
-export const getActiveUserName = (): string => store.getState().UI.get('user')?.metadata?.name;
+export const getActiveUserName = (): string => store.getState().core.user?.metadata?.name;
```

Comment:
I trust you'll find all of these and use the selector and I don't have to call them all out 🙂 ; even if these functions really should never exist.